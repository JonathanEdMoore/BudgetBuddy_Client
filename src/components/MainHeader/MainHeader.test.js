import React from 'react'
import ReactDOM from 'react-dom'
import MainHeader from './MainHeader'
import { BrowserRouter } from 'react-router-dom'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe('MainHeader smoke test', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<BrowserRouter><MainHeader /></BrowserRouter>, div)
    ReactDOM.unmountComponentAtNode(div)
  })
})

describe('MainHeader snapshot test', () => {
  it('renders the UI as expected', () => {
    const wrapper = shallow(<MainHeader />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})