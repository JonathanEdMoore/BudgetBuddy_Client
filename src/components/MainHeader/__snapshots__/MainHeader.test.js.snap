// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainHeader snapshot test renders the UI as expected 1`] = `
<div>
  <header
    role="banner"
  >
    <Link
      to="/"
    >
      <h1>
        BudgetBuddy
      </h1>
    </Link>
    <div>
      <SignupButton />
      <span
        className="Hyph"
      >
         - 
      </span>
      <LoginButton />
    </div>
    <Link
      to="/home"
    >
      Home
    </Link>
  </header>
</div>
`;
